MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -2872,9 +2906,13 @@ static uint32_t mvkGetEntryProperty(io_registry_entry_t entry, CFStringRef prope
 	pdScalarLayoutFeatures.sType = VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_SCALAR_BLOCK_LAYOUT_FEATURES_EXT;
 	pdScalarLayoutFeatures.pNext = &pdTexelBuffAlignFeatures;
 
+	VkPhysicalDeviceSamplerYcbcrConversionFeatures pdSamplerYcbcrConversionFeatures;
+	pdScalarLayoutFeatures.sType = VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_SAMPLER_YCBCR_CONVERSION_FEATURES;
+	pdScalarLayoutFeatures.pNext = &pdScalarLayoutFeatures;
```

Comment:
Copy-pasto `pdScalarLayoutFeatures`. Just so we're clear, the `pNext` pointer still needs to point to it; it's just you're writing to the wrong structure here.

Reply:
Good catch!